strings/piglatinGame.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Arrays;
+public class piglatinGame 
+{

```

Comment:
Add javadoc comments. You can refer to [Java-Documentation Comments](https://www.tutorialspoint.com/java/java_documentation.htm). Also refer to this file to analyse the format of code [DecimalToAnyBase.java](https://github.com/TheAlgorithms/Java/blob/Development/src/main/java/com/conversions/DecimalToAnyBase.java)